intro-HPC/ch_FAQ.tex

```diff
@@ -1,13 +1,18 @@
 \chapter{Frequently Asked Questions}
 \label{ch:faq}
 
-\ifgent
-% For other institutions: see https://github.com/hpcugent/vsc_user_docs/issues/246
 \section{When will my job start?}
 
-See the explanation about how jobs get prioritized in subsection~\ref{subsec:priority}.
+\ifgent
+See the explanation about how jobs get prioritized in \autoref{subsec:priority}.
+\else
+\ifbrussel
```

Comment:
No blocker for this PR, but can you check whether it's feasible to support something like `\ifgent or \ifbrussel`?

Reply:
This is not possible using our current setup.

Ideally, we want to add a macro with a name like `\ifcheckjob` (or whatever property it is Gent and Brussel have in common that makes checkjob unavailable). That way, we don' lose the meaning of the conditional and make it easy to change it if Leuven or Antwerpen decides to disable these commands.

Reply:
That's worth considering, but we need to avoid that we start introducing a lot more context-specific macros...

In this case, it makes sense though. `\ifusermoabcommands` maybe makes sense (`checkjob`, `showstart`, etc. are all command that talk to the MOAB job scheduler, and in Ghent & Brussels they're not available to users).

Reply:
Can you open a dedicated issue on this to look into it?